packages/lit-element/package.json

```diff
@@ -255,9 +255,9 @@
     "!/development/test/"
   ],
   "dependencies": {
-    "@lit-labs/ssr-dom-shim": "^1.0.0",
+    "@lit-labs/ssr-dom-shim": "^1.1.0",
     "@lit/reactive-element": "^1.3.0",
-    "lit-html": "^2.2.0"
+    "lit-html": "^2.7.0"
```

Comment:
Do we need this bump?

Reply:
It was done automatically by changesets. I don't think there's any specific code changes in `lit-element` in this release that directly depends on the new changes in `lit-html`, but for users of SSR, they'd need both of these to be the latest version.

With our suggested method of using Lit being via the `lit` package which'll depend on the latest of both of these, I don't think it hurts to make sure these core packages depend on the latest version available to encourage updating in lock-step.

Reply:
that's true It `lit` depends on latest, `lit-element` might as well too